FLEDGE.md

```diff
@@ -179,6 +179,10 @@ const myAuctionConfig = {
                         'https://www.another-buyer.com': 200,
                         '*': 150,
                         ...},
+  'perBuyerBidGenerationTimeouts': {'https://www.example-dsp.com': 500,
```

Comment:
Another API-ness nit: the auction configuration already contains a number of per-buyer dictionaries. I wonder if it would make sense to add some form of a buyer-level configuration object (which may include various timeouts and other constraints, as well as per-buyer signals), and then add a single per-buyer-origin dictionary, with values containing such a buyer-level configuration object. I understand there's a backwards compatibility issue if this route were taken, though FLEDGE may be in an early enough testing phase to be able to afford the change.

Reply:
We've been thinking that we might make just that change before standardization, so as not to break folks during field trials/experimentation.